test/src/main/java/org/apache/accumulo/test/ScanConsistencyIT.java

```diff
@@ -79,12 +79,58 @@
 public class ScanConsistencyIT extends AccumuloClusterHarness {
 
   private static final Logger log = LoggerFactory.getLogger(ScanConsistencyIT.class);
+  private static boolean inTestingContext;
+
+  public static void main(String[] args) {
+    /**
+     * @formatter:off
+     * Note: In order to run main,
+     * 1) Build the project
+     * 2) Copy the accumulo test jar (in /test/target/) to your accumulo installation's
+     * lib directory*
+     * Now, this can be run with
+     * "accumulo org.apache.accumulo.test.ScanConsistencyIT <props-file> <tmp-dir> <table>"
+     *      <props-file>: An accumulo client properties file
+     *      <tmp-dir>: tmpDir field for the TestContext object
+     *      <table>: The name of the table to be created
```

Comment:
When running standalone, do we want a new table to be created? Or do we want to be able to pass in an existing table?

Reply:
The test would need an empty table if it existed.  Would probably be good to create the table and have the test fail if it exists.  The call to create table should fail if it exists, so should not be anything to do there.  Could delete the table when the test is done.